Operational-Procedures/Main-Telescope/ComCam-Master-Calibrations-Procedure/comcam-master-calibrations-procedure.rst

```diff
@@ -0,0 +1,212 @@
+.. |author| replace:: *Andrés A. Plazas Malagón*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *none*
+
+.. _ComCam-Master-Calibrations-Procedure:
+
+###########################
+ComCam Master Calibrations Generation Procedure
+###########################
+
+.. _ComCam-Master-Calibrations-Procedure-Overview:
+
+Overview
+========
+
+This procedure describes how to call scripts to produce master calibrations for LSSTComCam from the LSST Operations and Visualization Enviroment (LOVE) at the Summit :ref:`operational environment <Observing-Interface-Operational-Environments>`. 
+
+The script will have the option to: 
+
+- command LSSTComCam to take a number of calibrations frames:
+   - biases
+   - darks 
+   - flats
+- call the corresponding Rubin Science Pipelines calibration generation pipetask to produce biases, darks, flats, defects and Photon Transfer Curves (PTCs) via the OCS-Controlled Pipeline System (OCPS),
+- verify the resulting calibration (see the package `cp_verify`_ and `DMTN-101`_),
+- certify the resulting calibration with a given range of validity dates.
+
+The script currently has the option (via the `script_mode` parameter in the configuration options) to:
+
+- take only biases, 
+- take biases and darks, and 
+- take biases, darks, and flats. 
+  
+These options are constrained by the fact that one calibration depends on the existence of the previous one (i.e., to build a dark, a bias is necessary, and to build a flat, a dark and a bias are necessary).
+
+If desired, defects can be constructed from darks and flats (in which case  `script_mode` must be ``BIAS_DARK_FLAT``), and a PTC per detector per amplifier constructed form the flats. Note that the PTC assumes that a sequence of flat pairs has been taken, each pair taken at the same exposure time (in order to produce a PTC, `script_mode` must be ``BIAS_DARK_FLAT`` too).
+
+For more information about calibrations production (including verification and certification), please consult the `Constructing Calibrations documentation`_.
+
+.. _cp_verify: https://github.com/lsst/cp_verify
+.. _DMTN-101: https://dmtn-101.lsst.io/
+.. _Constructing Calibrations documentation: https://lsst.ncsa.illinois.edu/~czw/pipelines_lsst_io/_build/html/modules/lsst.cp.pipe/constructing-calibrations.html 
+
+.. _ComCam-Master-Calibrations-Procedure-Prerequisites:
+
+
+Prerequisites
+=============
+
+- You should be logged into the LSST Operations and Visualization Enviroment (LOVE) at the Summit :ref:`operational environment <Observing-Interface-Operational-Environments>`.
+- The script assumes (and checks) that the ``LSSTComCam`` and ``OCPS`` components are all ``ENABLED``, and that the latter has been ``ENABLED`` with the configuration of ``LSSTComCam``. The instrument and the ``OCPS`` can be enabled, for example, from a notebook: 
```

Comment:
Same comment as I made in the LATISS one, I think we should have separate procedures for how to do this. In the case of ComCam, we are not preparing the telescope for flat field, so we can skip it for now.